File: CS50x/sentimental-mario-more/mario.py
import cs50

height = cs50.get_int("height: ")

while height < 1 or height > 8:
    height = cs50.get_int("height: ")


for i in range(height): #높이
    print(" " * (height-i-1), end="")
    print("#" * (i+1) , end="")
    print("  " , end="")
    print("#" * (i+1))

# for loop가 i = 0에서 시작하다보니 처음에 아무것도 없는 newline만들고 시작함.

File: CS50x/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    do
    {
    n = get_int("height(1~8): ");
    }

    while(n>8 || n<1);

    for(int i = 0, a=1; i < n; i++)
    {
        for(int h = 0; h < n-a; h++)
        {
        printf(" ");
        }

        for(int j = 0; j < a; j++)
        {
        printf("#");
        }

        printf("  ");

        for(int j = 0; j < a; j++)
        {
        printf("#");
        }

    printf("\n");

    a++;
    }
}